apisx/v1alpha1/xlistenerset_types.go

```diff
@@ -167,8 +167,6 @@ type ListenerEntry struct {
 	// +optional
 	//
 	// +kubebuilder:default=0
-	// +kubebuilder:validation:Minimum=0
-	// +kubebuilder:validation:Maximum=65535
```

Comment:
This and the related validation below is very much intentional and the removal of these tags seems to be removing corresponding validation on the CRD, so I'd like to leave these in place.